reference/generators/cmake_find_package.rst

```diff
@@ -55,8 +55,56 @@ utilities exported by the package available for consumers just by setting `find_
 Moreover, this also adjusts `CMAKE_MODULE_PATH` and `CMAKE_PREFIX_PATH` to the values declared by the package in ``cpp_info.buildirs``, so
 modules in those directories can be found.
 
-Target in Find<PKG-NAME>.cmake
-------------------------------
+Components
+++++++++++
```

Comment:
I would expect previous variables (`<PKG-NAME>_INCLUDE_DIRS`, `<PKG-NAME>_SYSTEM_LIBS`,...) to be available too, so _"if using components these variables will be added to the ones listed before:"_: